gcc/rust/expand/rust-macro-builtins.cc

```diff
@@ -105,6 +105,22 @@ std::unordered_map<
     {"test_case", MacroBuiltin::sorry},
     {"global_allocator", MacroBuiltin::sorry},
     {"cfg_accessible", MacroBuiltin::sorry},
+    /* Derive builtins do not need a real transcriber, but still need one. It
+       will however never be called since builtin derive macros get expanded
+       differently, and benefit from knowing on what kind of items they are
+       applied (struct, enums, unions) rather than receiving a list of tokens
+       like regular builtin macros */
+    {"RustcEncodable", MacroBuiltin::dummy},
```

Comment:
I wonder should it be called MacroBuiltin::builtinDerive maybe. Hmm not sure I like that either lol

Reply:
I think `MacroBuiltin::proc_macro_builtin` would be better maybe, since we'll need to apply it to macros like `#[test]` too

Reply:
@philberty let me know what you think about that name and I'll merge this :)